modules/postgres/postgres.go

```diff
@@ -36,11 +37,14 @@ func (c *PostgresContainer) ConnectionString(ctx context.Context, args ...string
 	}
 
 	extraArgs := ""
-	for _, arg := range args {
-		extraArgs += " " + arg
+	if len(args) >= 1 {
+		extraArgs = args[0]
+	}
+	for _, arg := range args[1:] {
```

Comment:
this is to skip the first arg since we already added it to the string (to avoid ?&fo=var which is not valid)

Reply:
I think you can simplify it with strings.Join:

https://go.dev/play/p/6AEmqgAXfrq?v=goprev